packaging/rpm/flightctl.spec

```diff
@@ -401,7 +407,7 @@ echo "Flightctl Observability Stack uninstalled."
 
 %prep
 %goprep -A
-%setup -q %{forgesetupargs}
+%setup -q %{forgesetupargs} -n flightctl-0.10.0~main~276~gdf8aac99
```

Comment:
1. It shoudln't be a hardcoded version here
2. what was wrong with a default %{name}=%{version}?